ftag/flavour.py

```diff
@@ -16,7 +16,9 @@ class Flavour:
 
     @property
     def px(self) -> str:
-        return f"p{self.name.removesuffix('jets')}"
+        if self.name.endswith("jets"):
+            return f"p{self.name[: -len('jets')]}"
```

Comment:
what about simple `str.replace(a,b)`? or is this not safe enough to only remove a suffix?